### PR TITLE
(#346) Sample for custom SQL statement

### PR DESCRIPTION
**Pull request by adrianhall**

### CODE REVIEW

**Comment by mamaso** [comment]
Looks good

File: samples/custom-api-sql-stmt/api/completeall.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var api = {
+    get: (request, response, next) => {
+        var query = {
+            sql: 'UPDATE TodoItem SET complete=@completed',
+            parameters: [{
+                completed: request.params.completed
```
Comment (danderson00) [inline samples/custom-api-sql-stmt/api/completeall.js]: This is not correct. Please see the format for parameters in the original issue.

Comment (mamaso) [inline samples/custom-api-sql-stmt/api/completeall.js]: Good catch Dale, should be [{ name: 'completed', value: request.params.completed }] I believe..

Comment (adrianhall) [inline samples/custom-api-sql-stmt/api/completeall.js]: Good catch - PR #361 just submitted since this one was already merged.

**Pull request merged**